### PR TITLE
Rails forms

### PR DESCRIPTION
**Pull request by cvalencia1991**
# in this milestone, I did the following activities 

- I add the like button to add the like for each post 
- add the form for the Post 
- add the form for the comment 
- Fix Rubocop and linters Errors

### CODE REVIEW

File: app/controllers/likes_controller.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class LikesController < ApplicationController
+  def create
+    @like = Like.new(:user_id, :id)
+    if @like.save
```
Comment (codepantha) [inline app/controllers/likes_controller.rb]: - [ ] The functionality to like a post is not working as expected. Currently clicking on the button simply refreshes the page and does not actually like the post. Please try to fix this 🙏 
**HINT:**  The `postid` is unaccounted for so which post are you trying to add a like to?👍 
image

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

## H @cvalencia1991 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Descriptive PR
✔️ Good commit history
✔️ Added forms 📌 
✔️ No linter errors

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/likes_controller.rb (hunk 1 above)
Comment (cvalencia1991) [reply app/controllers/likes_controller.rb]: hi @codepantha  im trying to add a like for the comment that is for the post, but It couldn't deploy it, could you please see what is the error ??

**Review by Danieldotcomcoder — APPROVED** [review]
Hello @cvalencia1991 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 👍 👍 😄 👏 

Congratulations! 🎉 🎉 🎉 

## Highlights
- Professional Readme File ✔✔
- Linters are setup correctly ✔✔
- Clear and nice code ✔✔

comp-2
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**